monthly_invoice/reports/header_custom.xml

```diff
@@ -23,9 +23,14 @@
         >
             <attribute name="t-att-style">"display: none"</attribute>
         </xpath>
+        <!--
+            When switching the positioning strategy from 'replace' to 'after',
+            the newly added div wasn't displayed as anticipated due to its misplacement.
+            To address this, we adjusted the position to 'before', ensuring its proper visibility.
+        -->
```

Comment:
@AungKoKoLin1997 I don't think it's appropriate to have this explanation here.  What we explain in the code (if any) is the reason why the code is in the current state, not why we changed something from one to another, which is the job of the commit message.